docs/en/user_guides/visualization.md

```diff
@@ -76,4 +76,89 @@ we can also run the following command to view them in TensorBoard:
 tensorboard --logdir work_dirs/test_visual/20220810_115248/vis_data
 ```
 
-If you would like to know more visualization usage, you can refer to [visualization tutorial](https://mmengine.readthedocs.io/en/latest/advanced_tutorials/visualization.html) in mmengie.
+### Visualizer Data Samples during Model Testing or Validataion
+
+Noted that `SegVisualizationHook` is used to visualize validation and testing process prediction results, defined [here](../../../mmseg/engine/hooks/visualization_hook.py).
+
+If you want to visualize a single data sample, a pair of input image and its ground truth in the dataset are necessary.
+You could prepare them on your own or download examples below by following commands:
+
+<div align=center>
+<img src="https://user-images.githubusercontent.com/24582831/189833109-eddad58f-f777-4fc0-b98a-6bd429143b06.png" width="70%"/>
+</div>
+
+```shell
+wget https://user-images.githubusercontent.com/24582831/189833109-eddad58f-f777-4fc0-b98a-6bd429143b06.png --output-document aachen_000000_000019_leftImg8bit.png
+wget https://user-images.githubusercontent.com/24582831/189833143-15f60f8a-4d1e-4cbb-a6e7-5e2233869fac.png --output-document aachen_000000_000019_gtFine_labelTrainIds.png
+```
+
+Then you can find their local path and use the scrips below to visualize:
+
+```python
+import mmcv
+import os.path as osp
+import torch
+# `PixelData` is data structure for pixel-level annotations or predictions defined in MMEngine.
+from mmengine.structures import PixelData
+
+# `SegDataSample` is data structure interface between different components
+# defined in MMSegmentation, it includes ground truth, prediction and
+# predicted logits of semantic segmentation.
+from mmseg.structures import SegDataSample
+from mmseg.visualization import SegLocalVisualizer
+
+out_file = 'out_file_cityscapes'
+save_dir = './work_dirs'
+
+image = mmcv.imread(
+    osp.join(
+        osp.dirname(__file__),
+        './aachen_000000_000019_leftImg8bit.png'
+    ),
+    'color')
+sem_seg = mmcv.imread(
+    osp.join(
+        osp.dirname(__file__),
+        './aachen_000000_000019_gtFine_labelTrainIds.png'  # noqa
+    ),
+    'unchanged')
+sem_seg = torch.from_numpy(sem_seg)
+gt_sem_seg_data = dict(data=sem_seg)
+gt_sem_seg = PixelData(**gt_sem_seg_data)
+data_sample = SegDataSample()
+data_sample.gt_sem_seg = gt_sem_seg
+
+seg_local_visualizer = SegLocalVisualizer(
+    vis_backends=[dict(type='LocalVisBackend')],
+    save_dir=save_dir)
+
+# The meta information of dataset usually includes `classes` for class names and
+# `palette` for visualization color of each foreground.
+# It is usually defined in corresponding class of dataset such as './mmseg/datasets/cityscapes.py'
```

Comment:
I suggest use this file https://github.com/open-mmlab/mmsegmentation/blob/1.x/mmseg/utils/class_names.py

All class names and palettes are defined in the file ...